src/kvstore/test/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ add_executable(
     part_test
     PartTest.cpp
     $<TARGET_OBJECTS:kvstore_obj>
+    $<TARGET_OBJECTS:dataman_obj>
```

Comment:
Do we still need dataman_obj after hbase_obj removed ?

Reply:
According to the design #179 , when using HBase to store data,  need to perform encode and decode operations, so it needs to use the **dataman** module. We can't remove it here.

Reply:
Yes, you are right, I will fix it soon.